anuvaad-etl/anuvaad-extractor/file_translator/etl-file-translator/services/docx_transform.py

```diff
@@ -224,19 +227,25 @@ def iterate_paras(self, document):
 
     def add_new_paragraph_to_json(self, para):
         try:
-            self.para_count += 1
-
-            json_para = self.generate_json_for_para(para=para, file_id=self.file_id, para_idx=str(self.sequence_para_index))
-
-            words_no = common_obj.word_count(json_para.get('text'))
-            self.word_count += words_no
+            sub_para_counter = -1
+            for idx, subpara in enumerate(common_obj.get_runs(para, para_obj=True)):
```

Comment:
i think you don't need a extra variable sub_para_counter,instead you can use idx

Reply:
Got it, will remove the extra variable and replace it with idx itself.